blurring_as_a_service/move_data_pipeline/components/move_files.py

```diff
@@ -16,7 +16,7 @@
 )
 settings = BlurringAsAServiceSettings.set_from_yaml(config_path)
 aml_experiment_settings = settings["aml_experiment_details"]
-
+IMG_FORMATS = ('bmp', 'dng', 'jpeg', 'jpg', 'mpo', 'png', 'tif', 'tiff', 'webp', 'pfm')  # image suffixes defined in YOLOv5
```

Comment:
I'm really not a fan of global constants. If it's defined in Yolo and depending on that, what about placing it somewhere there? Maybe not for now but for a future refactoring.

Reply:
Agree. Will make a task refactoring pipeline code with this one in it